spec/jekyll_seo_tag_spec.rb

```diff
@@ -218,18 +218,6 @@
     end
   end
 
-  context "with site.github.url" do
-    let(:github_namespace) { { "url" => "http://example.invalid" } }
-    let(:site) { make_site("github" => github_namespace) }
-
-    it "uses site.github.url to build the seo url" do
-      expected = %r!<link rel="canonical" href="http://example.invalid/page.html" \/>!
-      expect(output).to match(expected)
-      expected = %r!<meta property="og:url" content="http://example.invalid/page.html" />!
-      expect(output).to match(expected)
-    end
-  end
-
```

Comment:
Yep. This plugin can now rely on `absolute_url`, which will be based on `site.url`, which Pages will set... no need to bake proprietary namespaces into open source plugins. :smile: